apps/explorer/src/components/HomeMetrics/CurrentEpoch.tsx

```diff
@@ -30,64 +34,64 @@ export function CurrentEpoch() {
     }, [start]);
 
     return (
-        <NetworkStats bg="highlight" spacing="none">
-            <div className="flex flex-col gap-4">
-                <div className="space-y-4">
-                    <div className="flex flex-col gap-2">
-                        <Heading
-                            color="success-dark"
-                            variant="heading4/semibold"
-                        >
-                            Current Epoch
-                        </Heading>
-                        <Heading
-                            color="success-dark"
-                            variant="heading4/semibold"
-                        >
-                            {formatAmountParts(epoch)}
-                        </Heading>
-                    </div>
-
-                    {!progress && end ? (
-                        <div>
-                            <Text
-                                variant="pSubtitleSmall/normal"
-                                uppercase
-                                color="steel-darker"
+        <LinkWithQuery to={`/epoch/${epoch}`}>
```

Comment:
Wrapped the component in a link & added some placeholder UI copy text for when the data hasn't loaded yet